.renovaterc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": ["github>adobe/helix-shared"],
+  "ignoreDeps": [
+    "@spectrum-web-components/action-button",
```

Comment:
Can't we use something like `"@spectrum-web-components/*"`?

Reply:
Unfortunately not https://docs.renovatebot.com/configuration-options/#ignoredeps